### ISSUE TITLE
Dapp should not throw 404 error after refreshing

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-launch-dapp#478]

**Comment by dkent600** [comment]
Not a bug.  Problem was with fastcomet

**Comment by BartuCulha** [comment]
I'm receiving the same error against rinkeby but the site is working normal against mainnet. I guess we can move this card to accepted because it's related to fastcomet @dkent600

**Comment by dkent600** [comment]
Yes @BartuCulha , both rinkeby and mainnet sites are hosted by fastcomet
